reapit-config.d.ts

```diff
@@ -0,0 +1,140 @@
+declare namespace NodeJS {
```

Comment:
Re-declaring the NodeJS namespace is going to cause us problems because it is already declared in `@types/node` - need to re-think this approach